src/org/zaproxy/zap/extension/fuzz/httpfuzzer/ui/HttpFuzzerResultsTableModel.java

```diff
@@ -226,6 +227,14 @@ protected Object getCustomPrototypeValue(int columnIndex) {
 
         return results.get(row.intValue()).getPayloads();
     }
+    
+    public List<String> getHeaders() {
+    	List<String> ret = new ArrayList<>();
+    	for (Column x : Column.values()) {

```

Comment:
It should be using getColumnCount()/getColumnName(int) instead of the values of the enum. The enum does not contain the correct values used by this model (i.e. they are not in exact order, the names are different and it would be missing the custom columns).